packages/core/notification.js

```diff
@@ -2,22 +2,11 @@ import * as storage from '@octolinker/helper-settings';
 import { showNotification } from '@octolinker/user-interface';
 
 let notificationEl;
-const pkgVersion = require('./package.json')
-  .version.split('.')
-  .slice(0, -1)
-  .join('.');
-
-document.body.addEventListener('click', (event) => {
-  if (
-    event.target.classList.contains('js-hide-new-version') ||
-    event.target.classList.contains('js-flash-close-update-info')
-  ) {
-    storage.set('showUpdateInfo', false);
-    if (notificationEl) {
-      notificationEl.remove();
-    }
-  }
-});
+const releaseDescription = 'Test';
```

Comment:
Every time the new version notification shows it's prefixed with `Test - ` as you can see in my previous screen shots. Should this be changed/removed?

Reply:
The idea was rather than to make an API call to bake the release description into the release version itself. Obviously this requires updating this string, which can slip through easily. Let me see if I can find a way to fail the build in such a case.